app/controllers/internal/invitations_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Internal
+  class InvitationsController < Internal::ApplicationController
+    layout "internal"
+
+    def index
+      @invitations = User.where(registered: false).page(params[:page]).per(50)
+    end
+
+    def new; end
+
+    def create
+      email = params[:user][:email]
```

Comment:
**suggestion (non-blocking):** I prefer using

```ruby
params.dig(:user, :email)
```

as it reads better and is `nil` safe if a key is missing.